### PR TITLE
Don't generate TrickPlay images for files that don't exist

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

**Review by Shadowghost — APPROVED** [review]

File: Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs [hunk 1]
```diff
@@ -121,6 +121,13 @@ public async Task RefreshTrickplayDataAsync(Video video, bool replace, Cancellat
                     return;
                 }
 
+                var mediaPath = mediaSource.Path;
+                if (!File.Exists(mediaPath))
+                {
+                    _logger.LogWarning("Media not found: {Path}", mediaPath);
```
Comment (nielsvanvelzen) [inline Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs:127]: Please add the item id too

Comment (Bond-009) [reply Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs:127]: Id of the mediasource or the actual video ?

Comment (nielsvanvelzen) [reply Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs:127]: Both I guess. Logs should help with troubleshooting so I'd rather have a bit too much info in the log line.

File: Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs [hunk 2]
```diff
@@ -121,6 +121,13 @@ public async Task RefreshTrickplayDataAsync(Video video, bool replace, Cancellat
                     return;
                 }
 
+                var mediaPath = mediaSource.Path;
+                if (!File.Exists(mediaPath))
+                {
+                    _logger.LogWarning("Media source {MediaSourceId} not found at {Path} for item {ItemID}", mediaSource.Id, mediaPath, mediaSource.Id);
```
Comment (crobibero) [inline Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs:127]:
```suggestion
                    _logger.LogWarning("Media source {MediaSourceId} not found at {Path} for item {ItemID}", mediaSource.Id, mediaPath, video.Id);
```

**Review by crobibero — APPROVED** [review]

**Pull request merged**